docs/sidebars.js

```diff
@@ -56,7 +56,6 @@ module.exports = {
       "cli/manage-stake-accounts",
       "offline-signing",
       "offline-signing/durable-nonce",
-      "cli/usage",
```

Comment:
This builds if your run `docs/build.sh` which calls `build-cli-usage.sh` to create the `cli/usage.md` file.

Reply:
d'oh. fixed in new commit